test/features/toggle_fullrecord_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+feature 'ToggleFullrecord' do
+  before do
+    Capybara.current_session.driver.browser.clear_cookies
+  end
+
+  after do
+    Capybara.current_session.driver.browser.clear_cookies
+  end
+
+  test 'can toggle full record from app' do
```

Comment:
If I were being super picky I'd test whether the session variable got set. If I were thinking about the fact that the expected deployment time of this is ~2 months before we blow it away, I wouldn't care. Up to you :)

Reply:
It's in the back of my mind to move from session to cookies to persist features across sessions which FlipFlop can support so I'm going to leave this as checking the effects of the change rather than the change itself (for better or worse).

Reply:
OK then 🌈